app/views/content/blog/application-tips-from-a-teacher-training-provider.md

```diff
@@ -57,9 +57,13 @@ I can’t emphasise enough how important it is to avoid spelling mistakes. Make
 
 ## References
 
-And finally, we turn to references. Who you pick is really important. You will need two references. One could be a university or academic tutor and the second could be from a school or somewhere you have worked in a voluntary capacity, or it could be someone from your current employment.
+And finally, we turn to references. You'll need 2 referees.
 
-Whoever you choose, you should contact your references before you start writing your application so that they are aware that they will be contacted and that a swift response is important. If your references are not ready this can hold up your application. Read more about [choosing your referees.](/train-to-be-a-teacher/how-to-apply-for-teacher-training#choose-your-referees)
+Who you should pick as your referees will depend on your circumatances.
```

Comment:
```suggestion
The type of referee you need will depend on your circumstances.
```
I think you could probably improve this suggestion considerably - but "Who you should pick" feels a little clunky? More about need than 'pick'?